src/Components/Fetch.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+import Story from './Story';
+
+function Fetch ( { api } ) {
```

Comment:
This is all pretty clear from the code, but it wouldn't be bad to have a comment or two at times...Just to explain what you're doing and where you're putting info!